index.html

```diff
@@ -0,0 +1,133 @@
+  <link rel="stylesheet" href="style.css">
+</head>
+  
+  <body>
+
+    <header>
+      <div class = "banner">
+        <div class = "banner_button">
+          <a class = "logo" href="/"><img src = "image/logo.png"></a>
+        <a id = "login" href = "login.html">로그인</a>
+        </div>
+      </div>   
+    </header>
+
+    <main>
+      
+      <!-- Main Header -->
+      <section>
+        <div class = "header-container">
+          <div class = "header-content">
+            <div class = "header-text">
+              <h1 class = "header-title">
+                일상의 모든 물건을 <br>
+                거래해 보세요
+              </h1>
+              <a id = "explore-button" href = "itmes.html">구경하러 가기</a>
+            </div>
+            <img class = "header-image" src = "image/header.png">
+          </div>
+        </div>
+      </section>
+      
+      <!-- Main Body -->
+      <section class = "body-container">
+        <section class = "content-section">
+          <div class = "content-card">
+
+            <img src = "image/hotitem.png">
+
+            <div class = "content-text">
+              <span class = "content-label">Hot item</span>
+            
+              <h2 class = "content-title">
+                인기 상품을 <br>
+                확인해보세요
+              </h2>
+            
+              <p class = "content-description">
+                가장 HOT한 중고거래 물품을 <br>
+                판다 마켓에서 확인해 보세요
+              </p>
+
+            </div>
+          </div>
+        </section>
+        
+        <section class = "content-section">
+          <div class = "content-card">
+
+            <div class = "content-text">
+              <span class = "content-label">Search</span>
+              <h2 class = "content-title">
+                구매를 원하는<br>
+                상품을 검색하세요
+              </h2>
+              
+              <p class = "content-description">
+                구매하고 싶은 물품은 검색해서 <br>
+                쉽게 찾아보세요
+              </p>
+            </div>
+            
+            <img src = "image/search.png">
+
+          </div>
+        </section>
+        
+        <section class = "content-section">
+          <div class = "content-card">
+
+            <img src = "image/register.png">
+            
+            <div class = "content-text">
+              <span class = "content-label">Register</span>
+              <h2 class = "content-title">
+                판매를 원하는 <br>
+                상품을 등록하세요
+              </h2>
+              
+              <p class = "content-description">
+                어떤 물건이든 판매하고 싶은 상품을 <br>
+                쉽게 등록하세요
+              </p>
+            </div>
+          </div>
+        </section>
+        
+        <!-- Main Footer -->
+        <section class = "content-footer">
+            <div class = "cf-content">
+              <h3 class = "cf-text">
```

Comment:
💊 제안
한 레포안에서는 동일한 룰을 따르는 것이 좋습니다. 다른 classname들은 약어 사용이 없으시기도 하고 줄이지 않는것이
가독성면에서도 좋을 것 같아요. 따라서 content-footer로 사용하시는 것을 추천드려요~

```suggestion
            <div class = "content-footer-content">
              <h3 class = "content-footer-text">
```